src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,24 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article
+    className={classNames(
+      'TodoInfo',
+      {
+        'TodoInfo--completed': todo.completed,
+      },
+    )}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+    {todo.user && (
+      <UserInfo user={todo.user} />
+    )}
```

Comment:
it's line less then 80
```suggestion
    {todo.user &&  <UserInfo user={todo.user} />}
```